packages/vaadin-select/vaadin-select.d.ts

```diff
@@ -0,0 +1,2 @@
+export * from './src/vaadin-select.js';
+export * from './src/interfaces';
```

Comment:
What is about `./src/interfaces`? Shouldn't it forward imports as well as `./src/vaadin-select.js`?

Reply:
Oh, there is no such file at all. Then, I suppose, the export should be removed.

Reply:
Added an export from the new package so it doesn't break someone's code.